recipe/meta.yaml

```diff
@@ -10,8 +10,10 @@ source:
   sha256: 8e19250654a204af0858408b55dc78941382a8c824bf38fc7f2a95ca6e16d7a0
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and vc<14]
+  run_exports:
+    - {{ pin_subpackage("uriparser", max_pin="x.x.x") }}
```

Comment:
Given the abi stability of this we can safely pin to just `x`

Reply:
I know historically it's been ABI stable, but I couldn't find any guarantees and it is `0.x`.

Reply:
From offline conversation: package is 15+ years old, likely never going to 1.x and has been ABI stable. Changing to just `x`